app/queries/servers/thread.ts

```diff
@@ -123,3 +125,42 @@ export const queryUnreadsAndMentionsInTeam = (database: Database, teamId: string
         }),
     );
 };
+
+export async function queryNewestThreadInTeam(
```

Comment:
Can we add param `limit` and adding `Q.take(limit)` to `queryThreadsInTeam`? I know it's getting overloaded with params 😅 We can have this function but it can reuse the `queryThreadsInTeam` internally.